cmd/agent/main.go

```diff
@@ -1,188 +1,7 @@
 package main
 
-import (
-	"fmt"
-	"math/rand"
-	"net/http"
-	"os"
-	"os/signal"
-	"reflect"
-	"runtime"
-	"strconv"
-	"sync"
-	"syscall"
-	"time"
-)
-
-var pollInterval = 2 * time.Second
-var reportInterval = 10 * time.Second
-var contentType = "text/plain"
-var srvAddr = "http://127.0.0.1:8080"
+import "YP_GO_devops/internal/agent"
```

Comment:
вообще принято называть модули аналогично адресу до репозитория, то есть выглядеть должно примерно так
```suggestion
import "github.com/dcaiman/YP_GO/internal/agent"
```
Соответственно в go.mod пакет так же должен быть github.com/dcaiman/YP_GO. То есть когда ты инициировал модули можно было вызвать `go mod init github.com/dcaiman/YP_GO`

Reply:
ну и соответственно так как название репозитория это по сути название пакета, то не стоит использовать подчеркивания и верхний регистр https://go.dev/blog/package-names Собственно сейчас это не сильно критично, но знать стоит)